docs/integrators/snapshotless-observing-squad.md

```diff
@@ -24,10 +24,33 @@ The nodes started with the flag `--operation-mode snapshotless-observer` will no
 
 ### Creating a Snapshotless Observing Squad from scratch
 
-If you choose to install a snapshotless observing squad from scratch, you should follow the instruction from the [observing squad section](/integrators/observing-squad) and remember to add in the `variables.cfg` file the operation mode in the node's extra flags definition:
+If you choose to install a snapshotless observing squad from scratch, you should follow the instruction from the [observing squad section](/integrators/observing-squad) and remember to do the following updates:
+* add in the `variables.cfg` file the operation mode in the node's extra flags definition:
 ```
 NODE_EXTRA_FLAGS="-log-save -operation-mode snapshotless-observer"
 ```
+* update the proxy config created by the scripts(found at `$CUSTOM_HOME/elrond-proxy/config/config.toml`). `IsSnapshotless = true` should be added to each observer, as follows:
```

Comment:
this step is optional. if there aren't any snapshotless observers defined in config, proxy will use regular observers. so basically a 'default' proxy with only snapshotless observers behind will be a snapshotless proxy. 

so please state that this step is optional (useful when the integrators have 2 observing squads: a regular one and a snapshotless one. then yes, requests will be split accordingly).